packages/inferno-vnode-flags/src/index.ts

```diff
@@ -17,8 +18,14 @@ const enum VNodeFlags {
 	Void = 1 << 12,
 
 	FormElement = InputElement | TextareaElement | SelectElement,
-	Element = HtmlElement | SvgElement | MediaElement | InputElement | TextareaElement | SelectElement,
+	Element = HtmlElement |
```

Comment:
I guess it would look different to union types:
```
type Element = HtmlElement
  | SvgElement
  | MediaElement
  | InputElement
  | TextareaElement
  | SelectElement
```